### PR TITLE
blog: add sep monthly-report

### PR DESCRIPTION
**Pull request by Yilialinn**
This PR is for adding the Sep. monthly report of the Apache APISIX Community.
Need to be merged before Oct.1st.

### CODE REVIEW

File: blog/en/blog/2024/09/30/monthly-report.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+---
+title: "Monthly Report (September 01 - September 30)"
+keywords: ["Apache APISIX", "API Gateway", "Monthly Report", "Contributor"]
+description: Our monthly Apache APISIX community report generates insights into the project's monthly developments. The reports provide a pathway into the Apache APISIX community, ensuring that you stay well-informed and actively involved.
+tags: [Community]
+image: https://static.apiseven.com/uploads/2024/09/30/VW6dlYsu_sep-cover-en.png
+---
+
+> We have recently made some additions and improvements to specific features within Apache APISIX. The main improvements include adding `attach-consumer-label`, `ai-prompt-decorator`, and `ai-proxy` plugins, and supporting GCP Secret Manager, etc. For detailed information, please read the monthly report.
```
Comment (kayx23) [inline blog/en/blog/2024/09/30/monthly-report.md]:
```suggestion
> We have recently made some additions and improvements to specific features within Apache APISIX. The main improvements include the addition of `attach-consumer-label`, `ai-prompt-decorator`, and `ai-proxy` plugins, as well as support for GCP Secret Manager, among other enhancements. For detailed information, please read the monthly report.
```

File: blog/en/blog/2024/09/30/monthly-report.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+## Contributor Statistics
+
+![Apache APISIX Contributors List](https://static.apiseven.com/uploads/2024/09/30/LeOeANHk_Group%20427319848.png)
+
+![Apache APISIX New Contributors](https://static.apiseven.com/uploads/2024/09/30/BjHKV34C_sep-new-contributors.png)
+
+## Recent Feature Highlights
+
+1. [Add attach-consumer-label Plugin](https://github.com/apache/apisix/pull/11604)（Contributor: [dspo](https://github.com/dspo))
```
Comment (kayx23) [inline blog/en/blog/2024/09/30/monthly-report.md]: image

make them inline code, e.g.

```suggestion
1. [Add `attach-consumer-label` Plugin](https://github.com/apache/apisix/pull/11604)（Contributor: [dspo](https://github.com/dspo))
```

File: blog/en/blog/2024/09/30/monthly-report.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+This PR added the `gcp.lua` file to the original secret module, allowing users to store their secret information on GCP using the same reference method as before.
+
+5. [Add credential and Related Admin APIs](https://github.com/apache/apisix/pull/11601)（Contributor: [dspo](https://github.com/dspo))
+
+Credential is the object that holds the credential configuration of consumers. Consumers can use multiple credentials of different types, including `basic-auth`, `hmac-auth`, `jwt-auth`, and `key-auth`.
+
+6. [Refactor hmac-auth Plugin](https://github.com/apache/apisix/pull/11581)（Contributor: [Revolyssup](https://github.com/Revolyssup))
+
+The `hmac-auth` plugin has a lot of headers that need to be configured, which makes it confusing to use the plugin. This PR refactors the HMAC authentication plugin to improve its usability and compliance with RFC standards.
```
Comment (kayx23) [inline blog/en/blog/2024/09/30/monthly-report.md]: > ...has a lot of headers that need to be configured, which makes it confusing to use the plugin

reads a bit strange. Maybe just leave the second sentence?

File: blog/zh/blog/2024/09/30/monthly-report.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
+---
+title: "社区月报 (09.01 - 09.30)"
+keywords: ["Apache APISIX", "API 网关", "社区月报", "贡献者"]
+description: Apache APISIX 社区的月报旨在帮助社区成员更全面地了解社区的最新动态，方便大家参与到 Apache APISIX 社区中来。
+tags: [Community]
+image: https://static.apiseven.com/uploads/2024/09/30/InojHYaL_sep-cover-cn.png
+---
+
+> 最近，我们新增并改进了 Apache APISIX 的部分功能，包括新增 ai-proxy、ai-prompt-decorator、attach-consumer-label 插件和支持 GCP Secret Manager 等。有关更多功能新亮点，请阅读本期月报。
```
Comment (kayx23) [inline blog/zh/blog/2024/09/30/monthly-report.md]: inline code plugin names

File: blog/zh/blog/2024/09/30/monthly-report.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+
+## 贡献者统计
+
+![贡献者名单](https://static.apiseven.com/uploads/2024/09/30/LeOeANHk_Group%20427319848.png)
+
+![新晋贡献者](https://static.apiseven.com/uploads/2024/09/30/BjHKV34C_sep-new-contributors.png)
+
+## 近期亮点功能
+
+1. [新增 attach-consumer-label 插件](https://github.com/apache/apisix/pull/11604)（贡献者：[dspo](https://github.com/dspo))
```
Comment (kayx23) [inline blog/zh/blog/2024/09/30/monthly-report.md]: inline code plugin names. same for the below

File: blog/zh/blog/2024/09/30/monthly-report.md [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+
+凭证是持有消费者凭证配置的对象。消费者可以使用多种不同类型的凭证，包括 `basic-auth`、`hmac-auth`、`jwt-auth` 和 `key-auth`。
+
+6. [重构 hmac-auth 插件](https://github.com/apache/apisix/pull/11581)（贡献者：[Revolyssup](https://github.com/Revolyssup))
+
+`hmac-auth` 插件有许多需要配置的请求头，这使得使用该插件变得复杂。此 PR 对 HMAC 认证插件进行了重构，以提高其可用性并遵循 RFC 标准。
+
+7. [取消发放 JWT 令牌以增强安全性](https://github.com/apache/apisix/pull/11597)（贡献者：[dspo](https://github.com/dspo))
+
+出于安全原因，API 网关不应接受用户私钥的上传以发放 JWT 令牌，并且不再提供发放 JWT 令牌的 API。此 PR 移除了 /apisix/plugin/jwt/sign API。
```
Comment (kayx23) [inline blog/zh/blog/2024/09/30/monthly-report.md]:
```suggestion
出于安全原因，API 网关不应接受用户私钥的上传以发放 JWT 令牌，并且不再提供发放 JWT 令牌的 API。此 PR 移除了 `/apisix/plugin/jwt/sign` API。
```

File: blog/en/blog/2024/09/30/monthly-report.md [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+
+The `ai-proxy` plugin simplifies access to LLM providers and models by defining a standard request format that allows key fields in plugin configuration to be embedded into the request.
+
+4. [Add Support of GCP Secret Manager](https://github.com/apache/apisix/pull/11436)（Contributor: [HuanXin-Chen](https://github.com/HuanXin-Chen))
+
+This PR added the `gcp.lua` file to the original secret module, allowing users to store their secret information on GCP using the same reference method as before.
+
+5. [Add credential and Related Admin APIs](https://github.com/apache/apisix/pull/11601)（Contributor: [dspo](https://github.com/dspo))
+
+Credential is the object that holds the credential configuration of consumers. Consumers can use multiple credentials of different types, including `basic-auth`, `hmac-auth`, `jwt-auth`, and `key-auth`.
```
Comment (kayx23) [inline blog/en/blog/2024/09/30/monthly-report.md]: How about:

```suggestion
Credential is an entity used to store authentication configurations associated with consumers. A consumer can be associated with one or more credentials from a list of authentication plugins, including `basic-auth`, `hmac-auth`, `jwt-auth`, and `key-auth`.
```

**Review by kayx23 — COMMENTED** [review]
nitpicking. others lgtm.

**Review by kayx23 — APPROVED** [review]

**Review by shreemaan-abhishek — APPROVED** [review]

**Pull request merged**